### PR TITLE
MEPTS-893

### PR DESCRIPTION
**Pull request by rupansana12**

### CODE REVIEW

**Comment by ningosi** [comment]
Please resolve the conflicts

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java [hunk 1]
```diff
@@ -378,6 +378,12 @@ public CohortDefinition getAgeOnArtStartDate(
     return cd;
   }
 
+  /**
+   * Age should be calculated on Patient ART Start Date (Check Section A for the algorithm to define
+   * this date).
+   *
+   * @return CohortDefinition
+   */
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java:396]: Please include the parameters that are passed in the method in the javadoc, look at this as an example https://github.com/jembi/openmrs-module-eptsreports/blob/develop/api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java#L96-L106

Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java:396]: and if there are any tests written for any of the methods, they need to be documented there as well have a look at this method https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/api/PatientService.java#L72-L83

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java [hunk 2]
```diff
@@ -818,6 +818,13 @@ public static SqlCohortDefinition getMQ15NumI(
     return sqlCohortDefinition;
   }
 
+  /**
+   * All patients registered in Ficha Resumo (Encounter Type Id= 53) and marked as Transferred-in
+   * (“Transfer from other facility” concept Id 1369 = “Yes” concept id 1065) in TARV (“Type of
+   * Patient Transferred from” concept id 6300 = “ART” concept id 6276)
+   *
+   * @return SqlCohortDefinition
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java]: Just return the generic cohortDefinition

**Review by ningosi — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java (hunk 1 above)
Comment (rupansana12) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java:396]: Okay, I'll follow the examples.

**Review by cfaife — APPROVED** [review]

**Review by ningosi — APPROVED** [review]

**Pull request merged**